doc/voq/lag_hld.md

```diff
@@ -545,15 +546,49 @@ Shown below is the new attribute of SAI_OBJECT_TYPE_LAG object that is used for
 
 ## 8 Example Show Commands
 
+The system lag information are taken from **SYSTEM_LAG_TABLE** of **CHASSIS_APP_DB** from supervisor card.
+
+The show commands for system lags are under **show chassis** commands group as show below
+
+```
+admin@Linecard2:~$ show chassis -h
+Usage: show chassis [OPTIONS] COMMAND [ARGS]...
+
+  Chassis commands group
+
+Options:
+  -h, -?, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-lags       Show VOQ system lags information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
 ```
-admin@sonic:~$ show interfaces portchannel system
-Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available,
-       S - selected, D - deselected, * - not synced, L - local switch
-  No.  Switch ID    LAG Name      Protocol   Ports
------  ----------   -----------   ---------- --------------
-    1  0(L}         PortChannel1  SystemLAG  Slot1|Asic0|Ethernet1(S)  Slot1|Asic0|Ethernet2(S)
-    2  6            PortChannel1  SystemLAG  Slot2|Asic0|Ethernet1(S)  Slot2|Asic0|Ethernet2(S)
+### Syntax
 ```
+admin@Linecard2:~$ show chassis system-lags -h
+Usage: show chassis system-lags [OPTIONS] [SYSTEMLAGNAME]
+
+  Show VOQ system lags information
+
+Options:
+  -x, --asicname TEXT   Asic name
```

Comment:
why not use `-n` here ? similar to `show chassis system_ports`

Reply:
"-n" stands for "namespace". Only for system ports we get the system ports config info from databases from specific namespace. The "-x" option stands for "switch". This is getting info form chassis data bases in the supervisor card which is not in any specific namespace. The "-x" option is used for getting entries that have the given switch name in their keys.

Reply:
@vganesan-nokia. Thanks for the explanation. 
The option is little confusing for me. For example the output for the command `show chassis system-lags -x asic0` will display lags which have `asic0` in the key from all linecards. Is this correct ?

Would it simple if we provide option to filter on linecard iso of asic ?

Reply:
> @vganesan-nokia. Thanks for the explanation.
> The option is little confusing for me. For example the output for the command `show chassis system-lags -x asic0` will display lags which have `asic0` in the key from all linecards. Is this correct ?

**[@vganesan-nokia]** correct
> 
> Would it simple if we provide option to filter on linecard iso of asic ?

**[@vganesan-nokia ]**  Even though we filter on specific asic, the entries will be shown with the full key, which includes the 'hostname'.  So we can identify which asic is from which linecard (assuming 'hostname' has the string that identifies the linceard)

We can replace the "asicname" option. by filtering on 'hostname'. The asic name is the value of the mandatory attribute DEVICE_METADATA['localhost']['asic_name'].  So we have this filter on asic name. For 'linecard' we do not have such thing. However, in the multi slot chassis system we have a convention (not a written rule but kind of best practice) of using the mandatory attribute DEVICE_METADATA['localhost']['hostname'] to identify linecard/slot. This 'hostname' must be unique within the chassis. This is following the model of non chassis multi-asic box where the 'hostname' holds the name of the box. In chassis systems since each card acts as a multi asic box, this convention is followed. 

I feel that instead of replacing 'asicname' by 'hostname', we can have 'hostname' as additional option so that we can filter on entries for specific asic from a specific line card or entries from all asics of a specific line card. If you feel that this is too much filtering, I'll replace 'asicname' by 'hostname'.

Reply:
Fixed. Added option for filtering on host or linecard name.